datamade/openness-project-nmid
# Containerize application, remove unused edit_data and make_races management commands

## Description

This PR containerizes the application, to aid local development and in anticipation of migrating to Heroku from AWS.

### Testing instructions

- Pull down this branch, then follow the instructions in the updated README to run the application.
- Navigate to http://localhost:8000 and confirm you see the homepage with data, etc. Click around and confirm nothing breaks.
